### PR TITLE
Added current context view

### PR DESCRIPTION
**Pull request by eyal-solomon1**
* Added a small context identifier (was a bit confused on where it would be applied to)

Screenshot 2023-07-08 at 13 07 47

### CODE REVIEW

File: .DS_Store
Comment (sozercan) [inline .DS_Store:1]: looks like accidentally checked in, please remove

File: cmd/cli/kubernetes.go [hunk 1]
```diff
@@ -112,3 +109,29 @@ func getKubeConfig() string {
...
+
+	if err != nil {
+		return api.Config{}, err
+	}
+
+	return config, nil
+}
+
+func getCurrentContextName() (string, error) {
+	var kubeConfig string = getKubeConfig()
```
Comment (sozercan) [inline cmd/cli/kubernetes.go:128]:
```suggestion
	kubeConfig := getKubeConfig()
```

**Review by sozercan — CHANGES_REQUESTED** [review]
thanks! added minor comments and please address the lint errors, otherwise LGTM

**Comment by eyal-solomon1** [comment]
@sozercan fixed :)

**Review by sozercan — APPROVED** [review]
Thank you! LGTM

**Pull request merged**